tests/unit/error/merr_test.c

```diff
@@ -53,8 +53,12 @@ MTF_DEFINE_UTEST(hse_err_test, merr_test_1)
     ASSERT_EQ(0, strcmp(errinfo, "success"));
 
     rval = EINVAL;
+    // clang-format off
+    // - cannot let clang-format add or remove newlines
     err = merr(rval);
-    ASSERT_EQ(__LINE__ - 1, merr_lineno(err)); /* Hardcoded __LINE__-1 */
+    line = __LINE__ - 1; // line number were merr() was used
+    // clang-format off
+    ASSERT_EQ(line, merr_lineno(err));
```

Comment:
Leave a blank line before and after chunks like this, so that the chunk which begins with clang-format off and ends with clang-format on is clearly delineated...